File: spring-boot/Chat2Learn/src/main/java/com/project/chat2learn/service/model/response/RegisterResponse.java
package com.project.chat2learn.service.model.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RegisterResponse {

    private Long id;

    private String name;

    private String lastname;

    private String email;

    private String token;
}

File: chat2learnfe/lib/model/response/RegisterResponse.dart
class RegisterResponse {
  int? id;
  String? name;
  String? lastname;
  String? email;
  String? token;

  RegisterResponse({this.id, this.name, this.lastname, this.email, this.token});

  RegisterResponse.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    name = json['name'];
    lastname = json['lastname'];
    email = json['email'];
    token = json['token'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['name'] = this.name;
    data['lastname'] = this.lastname;
    data['email'] = this.email;
    data['token'] = this.token;
    return data;
  }
}
